core/variant/variant_parser.cpp

```diff
@@ -614,7 +614,7 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 			}
 
 			value = Plane(args[0], args[1], args[2], args[3]);
-		} else if (id == "Quat") {
+		} else if (id == "Quaternion") {
```

Comment:
Please add a compat check for `"Quat"` to preserve compatibility when loading scenes/resources. (Like in #49304.)

Reply:
Done.